requirements/requirements.txt

```diff
@@ -0,0 +1,5 @@
+word2number
+numpy
+ovos-core
```

Comment:
This can go in test_requirements, but shouldn't be in `requirements.txt` to support Mycroft core